web-ide/web_ide.py

```diff
@@ -168,24 +167,33 @@ async def on_shutdown():
     if dirpath.exists():
         shutil.rmtree(dirpath)
 
+async def export(q: Q):
+    shutil.make_archive('app', 'zip', '.', 'tmp_project')
+    zip_path = await q.site.upload(['app.zip'])
+    q.app.zip_path = f"{_server_adress}{zip_path[0]}"
```

Comment:
I guess this shall work as well, please verify.
```suggestion
    q.app.zip_path, = await q.site.upload(['app.zip'])
```